wordpress-mobile/WordPress-iOS
# Add enhanced screenshot localizations

Brings in the latest screenshot localizations.

The big change is in 0c20169b3949b707d2d4d42e9bff997253b68907 – it adds the additional keys.

Everything else is just running `bundle exec fastlane ios download_promo_strings` and committing the changes.

**To Test:**

1. Run `be fastlane update_appstore_strings`. Once the `.po` file has been created, try manually uploading it. You may want to make a small change to one of the source files in order to allow the script to generate an entirely new `.po` file.
2. Run `bundle exec fastlane ios download_promo_strings`. Assuming no translators have made changes recently, the repo should be clean – 0c20169b3949b707d2d4d42e9bff997253b68907 brings in all of the changes that would result from that (as of right now).

As an example of this working correctly, you can see 7b9d8e5a8d8af8c94cf83b858ab8187c43bfb356 – it correctly downloads metadata and saves it in the right format.
